Fixes Area Strangeness With Meta Xenobio

Paxilmaniac opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
All the areas at south xenobio on meta were somehow shifted up one, which means the south wall was actually in the space area and not xenobio, not anymore

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
The south wall of xenobio should probably be xenobio

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
fix: some strange area mistakes on meta xenobio
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

carda-sometimesknownas-paxilmaniac committed:
Merge remote-tracking branch 'upstream/master'

carda-sometimesknownas-paxilmaniac committed:
Merge remote-tracking branch 'origin/master'

carda-sometimesknownas-paxilmaniac committed:
Merge remote-tracking branch 'origin/master'

carda-sometimesknownas-paxilmaniac committed:
fixes some strange area fuckery that was going on in meta xenobio

It kinda looks like that entire area was shifted up one

uomo91 commented:
If you are on a quest to improve your PR balance, check out the monastery on Pubbystation, in the outside part there's a misplaced area tile.

uomo91 commented:
Why not just make that entire southern part xenobio maint? I see no reason not to.

Like this: 

![Untitled](https://user-images.githubusercontent.com/51800976/89934419-25b00b80-dbd6-11ea-8d20-2564a8e7574f.png)

carda-sometimesknownas-paxilmaniac committed:
strange area remnants

it seems when the xenobio slime killing chamber was moved, nobody bothered to change the area to xenobio maint
